File: 240927/007/007.py
class Class:
    def __init__(self, s, m, t):
        self.s = s
        self.m = m
        self.t = t

a, b, c = input().split()
c = Class(a, b, c)

print("secret code : " + c.s)
print("meeting point : " + c.m)
print("time : " + c.t)

File: 250103/007/007.cpp
#include <iostream>
using namespace std;

class Spy {
    public:
        string code; char loc; int time;

        Spy(string code, char loc, int time) {
            this->code = code;
            this->loc = loc;
            this->time = time;
        }
};

int main() {
    // Please write your code here.
    string code; char loc; int time;
    cin >> code >> loc >> time;
    Spy spy = Spy(code, loc, time);
    cout << "secret code : " << spy.code << endl;
    cout << "meeting point : " << spy.loc << endl;
    cout << "time : " << spy.time << endl;
    return 0;
}